docker_squash/image.py

```diff
@@ -932,19 +935,15 @@ def _path_hierarchy(self, path):
 
```

Comment:
Isn't the above comment description now wrong?

Reply:
Good call. I have modified the comment to better reflect the way the function now operates.